### ISSUE TITLE
[Feature request] improve search

### ISSUE THREAD

**Issue by Hikiru** [Memmy-App/memmy#795]
The current way search works seems to sort randomly, and a simple one word search can show many on semi-related subreddits with your actual target search result buried at the bottom. Even if it has thousands of members, it can have dozens of subreddits above it with a few hundred or even just a few people. 

My solution is to add some sorting and maybe filter options. Sort by relevancy (most closely matching name), subscriber count, activity, etc. I can’t think of filter options atm, especially since im not sure if communities can be marked as NSFW or not right now, but I would at least like the option to not check instance names when searching. For example, if I search “Lemmy” to just find a subreddit about Lemmy itself, it’ll include search results form instances with “Lemmy” in the instance name.

**Comment by eoddc5** [comment]
@gkasdorf im thinking the way search is working is how it’s intended. Any thoughts?

**Comment by eoddc5** [comment]
Yeah pretty sure there’s no search feature that’s native to memmy going on here. It’s just how searching on lemmy works

**Comment by marsara9** [comment]
This might solve the original commenters issue? https://github.com/Memmy-App/memmy/issues/758
